### ISSUE TITLE
PeAddSection.exe crashes if section name contains non-alphabetical characters

### ISSUE THREAD

**Issue by beatcracker** [Ge0/PeTools#1]
Example:

```none
PeAddSection.exe somexe.exe .new 0x60000020
```

**Comment by Ge0** [comment]
Hi!

It's been a while since I haven't touched this code, plus it's a bench of PoC.

Are you sure this is related to the section name? Plus, I think that my PoCs don't handle x64 architectures.

If you have more information about the bug, I'd be glad to take a look at it.

Cheers,

**Comment by beatcracker** [comment]
> Are you sure this is related to the section name?

Yep, as long as I use section name that contains only letters, it works.

>  Plus, I think that my PoCs don't handle x64 architectures.

Alas, yes. But it works fine for x86, aside from this issue.

> If you have more information about the bug, I'd be glad to take a look at it.

It crashes with exit code `0xc0000417`

![image](https://user-images.githubusercontent.com/7583050/34763416-2a70dd24-f5fc-11e7-82c7-056399692970.png)

Call stack:

![image](https://user-images.githubusercontent.com/7583050/34763491-775d4bae-f5fc-11e7-8842-8c4c20521800.png)

I'll try to provide more info later. Thank you!
